src/hotspot/share/gc/g1/g1FullGCCompactionPoint.hpp

```diff
@@ -31,40 +31,40 @@
 #include "utilities/pair.hpp"
 
 class G1FullCollector;
-class HeapRegion;
+class G1HeapRegion;
 class PreservedMarks;
 
 class G1FullGCCompactionPoint : public CHeapObj<mtGC> {
   G1FullCollector* _collector;
-  HeapRegion* _current_region;
+  G1HeapRegion* _current_region;
   HeapWord*   _compaction_top;
```

Comment:
Tidy indentation of `_compaction_top`.  My preference would be to remove extra whitespace before it,
rather than adding to (re)line up with new position of `_current_region`.